coforma/swift-tech-challenge
# Add logic for loading images from s3 per institution

# Summary

- Add s3 lib for pulling items from s3
- Add/update tests
- Update institutions processing to get images
  - S3 image if available
  - image icon if not found 

Fixes #80 

## Type of change

# How To Test
- run `yarn` to install the new dependency
- export AWS credentials in your terminal
- run `yarn dev`
- load `localhost:3000`
- verify each institution has a unique image

Extra mile:
- Modify the `institutions.ts` code such that it always returns the empty image icon
- Verify the display shows the icon (styling addressed separately)

# Checklist:
